case/src/main/kotlin/com/ritense/case/service/CaseDefinitionService.kt

```diff
@@ -48,44 +62,30 @@ class CaseDefinitionService(
     }
 
     @Throws(InvalidListColumnException::class)
-    fun createListColumn(caseDefinitionName: String, caseListColumnDto: CaseListColumnDto) {
-        validateListColumn(caseDefinitionName, caseListColumnDto)
-        caseDefinitionListColumnRepository.save(CaseListColumnMapper.toEntity(caseDefinitionName, caseListColumnDto))
-    }
+    fun upsertListColumn(
```

Comment:
The CREATE and UPDATE in this method, have their own blocks of code without overlapping code. This could have been split up into two methods